minecraft/src/main/java/org/spicord/velocity/VelocityJDADetector.java

```diff
@@ -0,0 +1,24 @@
+package org.spicord.velocity;
+
+import java.net.URL;
+
+import com.velocitypowered.api.plugin.PluginContainer;
+
+public class VelocityJDADetector {
+
+    public static void checkOtherJDA(SpicordVelocity spicordPlugin) {
+        for (PluginContainer plugin : SpicordVelocity.getProxyServer().getPluginManager().getPlugins()) {
+            if (plugin == spicordPlugin) {
```

Comment:
That condition will never be true as SpicordVelocity or any Velocity **plugin instance** does not extend PluginContainer.